src/game.zig

```diff
@@ -1263,13 +1316,21 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 			} else {
 				Player.super.pos[2] = box.min[2] - hitBox.max[2];
 			}
-
-			const damage: f32 = @floatCast(@round(@max((Player.super.vel[2]*Player.super.vel[2])/(2*gravity) - 7, 0))/2);
-			if(damage > 0.01) {
-				Inventory.Sync.addHealth(-damage, .fall, .client, Player.id);
+			var bounce = if(Player.isFlying.load(.monotonic)) 0 else collision.calculateBounce(.client, Player.super.pos, Player.outerBoundingBox);
+			if(KeyBoard.key("crouch").pressed) {
+				bounce *= 0.5;
 			}
+			if(bounce != 0.0 and Player.super.vel[2] < -4.0) {
```

Comment:
I think this cutoff feels a bit abrupt.

Also does this mean a material with 0.01 bouncyness will fully negate fall damage?
I think, maybe we should add a separate parameter for how much fall damage the player should get (specifiable as an offset to the height of the fall that you can survive)

Reply:
Oh yes, that is a great idea